src/debug.rs

```diff
@@ -0,0 +1,10 @@
+/// The native debug interface exposed to the ewasm contract. These functions are for testing
+/// purposes only. On a live VM, any bytecode trying to import these symbols will be rejected.
+extern "C" {
```

Comment:
I think adding a feature flag for conditional compilation of these features would be useful.

Reply:
Good idea!

Reply:
@s1na make sure that the conditional compilation flag is not on the interface declaration, but instead the `pub mod debug;` statement in the parent module.

Reply:
Yeah good idea Jake. Updated the code, is this what you meant?